issuer.go

```diff
@@ -6,6 +6,7 @@ package gabi
 
 import (
 	"crypto/rand"
+	"errors"
```

Comment:
In `gabi` and `irmago` we use `github.com/go-errors/errors` instead, which has the same API but adds stacktraces to errors. Please use that one instead.

Reply:
Updated. Want me to change https://github.com/privacybydesign/gabi/blob/master/proofs.go#L8 as well?